src/proto/TheOpenNetwork.proto

```diff
@@ -0,0 +1,67 @@
+// Copyright © 2017-2022 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+syntax = "proto3";
+
+package TW.TheOpenNetwork.Proto;
+option java_package = "wallet.core.jni.proto";
+
+enum WalletVersion {
+    WalletV3R1 = 0;
+    WalletV3R2 = 1;
+    WalletV4R2 = 2;
+};
+
+enum SendMode {
+    DEFAULT = 0;
+    PAY_FEES_SEPARATELY = 1;
+    IGNORE_ACTION_PHASE_ERRORS = 2;
+    DESTROY_ON_ZERO_BALANCE = 32;
+    ATTACH_ALL_INBOUND_MESSAGE_VALUE = 64;
+    ATTACH_ALL_CONTRACT_BALANCE = 128;
+};
+
+message Transfer {
+    // Wallet version
+    WalletVersion wallet_version = 1;
+
+    // Recipient address
+    string dest = 2;
+
+    // Amount to send in nanotons
+    uint64 amount = 3;
+
+    // Message counter (optional, 0 by default used for the first deploy)
+    // This field is required, because we need to protect the smart contract against "replay attacks"
+    // Learn more: https://ton.org/docs/develop/smart-contracts/guidelines/external-messages
+    uint32 seqno = 4;
+
+    // Send mode (optional, 0 by default)
+    // Learn more: https://ton.org/docs/develop/func/stdlib#send_raw_message
+    uint32 mode = 5;
+
+    // Expiration UNIX timestamp (optional, now() + 60 by default)
+    uint64 expire_at = 6;
```

Comment:
WalletV4R2 and lower wallets actually expect `expire_at` to be `uint32`.
At some moment, this should be updated, and protobuf will also be changed.
That's why we decided to change the protobuf now.